CMakeLists.txt

```
@@ -732,7 +732,7 @@ elseif(BUILD_GLES2)
 	set(CPACK_DEBIAN_PACKAGE_DEPENDS "${CPACK_DEBIAN_PACKAGE_DEPENDS},libgles2")
 endif()
 
-if(WIN32 AND NOT SKIP_LIBRARY AND NOT USE_EPOXY)
+if(WIN32 AND NOT LIBMGBA_ONLY AND NOT SKIP_LIBRARY AND NOT USE_EPOXY)
```

Might be more readable if it's turned into this:

```suggestion
if(WIN32 AND NOT (LIBMGBA_ONLY OR SKIP_LIBRARY OR USE_EPOXY))
```

But otherwise seems like a good idea.